### PR TITLE
Rebuild feelnc

### PR DESCRIPTION
**Pull request by cmonjeau**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by druvus** [comment]
Bioconda has moved to build against perl 5.26 so I think it would be better to fix dependencies that are not working. What issue did you have with perl 5.26 and feelnc?

File: recipes/feelnc/meta.yaml [hunk 1]
```diff
@@ -21,6 +21,7 @@ requirements:
     - perl-bio-featureio
 
   run:
+    - perl ==5.22.0.1
```
Comment (abretaud) [inline recipes/feelnc/meta.yaml]: my dear @cmonjeau, can you left it as it was before? I think just rebuilding should be enough
BUT you'll need to remove feelnc from the blacklist https://github.com/bioconda/bioconda-recipes/blob/master/build-fail-blacklist#L1964

File: recipes/feelnc/meta.yaml [hunk 2]
```diff
@@ -15,7 +15,6 @@ build:
 
 requirements:
   build:
-    - perl ==5.22.0.1
```
Comment (bgruening) [inline recipes/feelnc/meta.yaml]: perl should be given if perl is needed. This is a special case afaik for all scripting languages. Some for R or python.

Comment (abretaud) [reply recipes/feelnc/meta.yaml]: ok, good to know, I wasn't 100% sure

**Pull request merged**